tests/test_fairstep.py

```diff
@@ -88,3 +90,20 @@ def test_validation():
     step = FairStep(uri='http://www.example.org/step')
     with pytest.raises(AssertionError):
         step.validate()
+
+
+@pytest.mark.flaky(max_runs=10)
+@pytest.mark.skipif(nanopub_server_unavailable(), reason=SERVER_UNAVAILABLE)
+@patch('fairworkflows.nanopub_wrapper.publish')
+def test_modification_and_republishing(nanopub_wrapper_publish_mock):
+
+    preheat_oven = FairStep(uri='http://purl.org/np/RACLlhNijmCk4AX_2PuoBPHKfY1T6jieGaUPVFv-fWCAg#step', from_nanopub=True)
+    assert preheat_oven is not None
+    assert preheat_oven.is_modified is False
+    assert preheat_oven.publish_as_nanopub() is False
+
+    # Now modify the step description
+    preheat_oven.set_attribute(rdflib.DCTERMS.description, rdflib.term.Literal('Preheat an oven to 200 degrees C.'))
```

Comment:
```suggestion
    preheat_oven.description =  'Preheat an oven to 200 degrees C.'
```
This should do the same right? And is way more readable.

Reply:
Definitely better